cmd/zfs/zfs_main.c

```diff
@@ -2529,7 +2529,8 @@ userspace_cb(void *arg, const char *domain, uid_t rid, uint64_t space)
 		namelen = strlen(name);
 	}
 	nameidx = us_field_index("name");
-	if (namelen > cb->cb_width[nameidx])
+	if (nameidx >= 0 &&
+	    namelen > cb->cb_width[nameidx])
```

Comment:
Just a minor style issue then this LGTM.  There's enough space here where we can do both checks on a single line.  Please update both this check and the next one.

```
        if (nameidx >= 0 && namelen > cb->cb_width[nameidx])
```